openprescribing/frontend/price_per_unit/savings.py

```diff
@@ -89,57 +100,83 @@ def get_total_savings_for_org(date, org_type, org_id):
     """
     Get total available savings through presentation switches for the given org
     """
-    totals = get_total_savings_for_org_type(date, org_type)
     group_by_org = get_row_grouper(org_type)
+    totals = get_total_savings_for_org_type(
+        db=get_db(),
+        substitution_sets=get_substitution_sets(),
+        date=date,
+        group_by_org=group_by_org,
+        min_saving=CONFIG_MIN_SAVINGS_FOR_ORG_TYPE[org_type],
+        practice_group_by_org=get_row_grouper(CONFIG_TARGET_PEER_GROUP),
+        target_centile=CONFIG_TARGET_CENTILE,
+        practice_min_saving=CONFIG_MIN_SAVINGS_FOR_ORG_TYPE["practice"],
+    )
     offset = group_by_org.offsets[org_id]
     return totals[offset, 0] / 100
 
 
-def get_total_savings_for_org_type(date, org_type):
+def get_total_savings_for_org_type(
+    db,
+    substitution_sets,
+    date,
+    group_by_org,
+    min_saving,
+    practice_group_by_org,
+    target_centile,
+    practice_min_saving,
```

Comment:
Are these last four params really "global state"?

Reply:
No, they're not. They're more like "configuration values which affect the return value of the function" but I didn't have a convenient name for that. Maybe just "global state and configuration" would do actually.